quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -503,6 +594,87 @@ impl BbrCongestionController {
         (MIN_PIPE_CWND_PACKETS * self.max_datagram_size) as u32
     }
 
+    /// Updates the congestion window based on the latest model
+    fn set_cwnd(&mut self, newly_acked: usize) {
+        //= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.6.4.6
+        //# BBRSetCwnd():
+        //#   BBRUpdateMaxInflight()
+        //#   BBRModulateCwndForRecovery()
+        //#   if (!BBR.packet_conservation) {
+        //#     if (BBR.filled_pipe)
+        //#       cwnd = min(cwnd + rs.newly_acked, BBR.max_inflight)
+        //#     else if (cwnd < BBR.max_inflight || C.delivered < InitialCwnd)
+        //#       cwnd = cwnd + rs.newly_acked
+        //#     cwnd = max(cwnd, BBRMinPipeCwnd)
```

Comment:
I think we're missing this from what I can tell. Or is it handled in the `clamp` call? That does technically mean that the `BBR.packet_conservation` gets the clamping applied, even though this example code only does it in `!BBR.packet_conservation`

Reply:
Sounds good. Would it be worth adding a comment for that?